presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -168,19 +169,28 @@ else if (config.getPinGlueClientToCurrentRegion()) {
             }
         }
 
+        AWSCredentialsProvider credentialsProvider = createAwsCredentialsProvider(config);
+        asyncGlueClientBuilder.setCredentials(credentialsProvider);
```

Comment:
please inline `credentialsProvider` variable and (optionally) add a blank line before final return

Reply:
Sure